x-pack/plugins/security_solution/public/detections/containers/detection_engine/rules/use_rule_with_fallback.tsx

```diff
@@ -41,9 +43,20 @@ interface AlertHit {
   };
 }
 
-const fetchWithOptionslSignal = withOptionalSignal(fetchRuleById);
+// TODO: Is there an existing type that can be used here?
+interface RACRule {
+  kibana: {
+    alert: {
+      rule: {
+        parameters: {};
+      };
+    };
+  };
+}
+
```

Comment:
@marshallmain, is there a RACRule type that can be used here? I wasn't able to find one with the new `parameters` field.

Reply:
Checked with Marshall looks like not for now:

> Not yet unfortunately - there's a static type for the rule-type-specific params [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/common/detection_engine/schemas/request/rule_schemas.ts#L402), but we don't have a type for the full list of security-defined parameters in snake case. Since those types are for the API responses originally, they also include the alerting framework fields so it doesn't line up exactly with the `kibana.alert.rule.parameters` field


Will update TODO accordingly.. 👍